packages/clay-modal/src/Modal.tsx

```diff
@@ -44,6 +45,20 @@ const ClayModal: FunctionComponent<IProps> & {
 }: IProps) => {
 	const modalBodyElementRef = useRef<HTMLDivElement | null>(null);
 
+	warning(
+		observer !== undefined,
+		`You need to pass the 'observer' prop to ClayModal for everything to work fine, use the 'useModal' hook that exposes the observer.
+
+> const {observer} = useModal({...});
```

Comment:
I had to push it aside to not appear with spaces in the console. 🙂

Reply:
Since its a bit awkward to have in the middle of the component. Could you declare this message as a variable above?

Reply:
sure, np

Reply:
feel free to merge when updated